test/client-proxy/test-http-proxy-request.mjs

```diff
@@ -78,8 +78,6 @@ if (!common.isWindows) {
     REQUEST_URL: requestUrl,
     http_proxy: `http://localhost:${proxy.address().port}`,
     HTTP_PROXY: `http://localhost:${proxy2.address().port}`,
-  }, {
-    stdout: 'Hello world',
```

Comment:
These were copy-pasta from https://github.com/nodejs/node/pull/58980 - `runProxiedRequest` didn't take a second argument. So removed these bogus expectations to make way for the actual new second argument.